me.sanchezrodriguez.passes.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id" : "me.sanchezrodriguez.passes",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "passes",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "modules/python3-qrcode.json",
```

Comment:
this file is missing.

Reply:
I have just included the file.

Reply:
now, the preference is that the content of the file be inline in the main manifest as otherwise it's more complicated in maintenance, unless it's generated by a script.